main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/radiofrance/dib/cmd"
```

Comment:
Why don't we leave the main package there ? It's OK like that but I wonder why you did the change

Reply:
I used the `cobra` cli to bootstrap all cli commands, and this one also. By default, the cli will create this file in the main directory and the commands in `cmd`